### ISSUE TITLE
Notifications counter for group mentions no longer shown in accounts list

### ISSUE THREAD

**Issue by RememberTheAir** [telegramdesktop/tdesktop#24154]
### Steps to reproduce

1. open tdesktop, make sure to have at least two active accounts
2. in a group, ask someone to reply to one of your messages
3. switch to another account before someone replies
4. wait for someone to reply to your message
5. open the sidebar where tdesktop shows you the accounts list

### Expected behaviour

In the accounts list, next to the account name, a badge counter should appear, notifying you that someone mentioned you in a group (or, if the badge counter is already present, its counter should increase by 1)

### Actual behaviour

No badge counter is displayed

Up until a few versions ago, it used to appear next to the account's name _even for group chats mentions_. The android app (v8.5) will still show you a notification badge next to the mentioned account, so I assume it's the expected behavior and other clients still work this way (so I guess it's not a sever-side change in how notifications are broadcasted to clients)

### Operating system

Windows 10

### Version of Telegram Desktop

3.5.5

### Installation source

Static binary from official website

**Comment by RememberTheAir** [comment]
still an issue on v3.7+

**Comment by RememberTheAir** [comment]
Still a problem in 4.3, will it ever get fixed?

**Comment by RememberTheAir** [comment]
unstale

**Comment by RememberTheAir** [comment]
Still working this way. I wonder, was this change intended? Because it used to work as it does on the mobile app, but the behavior changed at some point, and now it's incosistent

**Comment by john-preston** [comment]
@RememberTheAir Please correct me, but the steps from the issue description should provide a counter anyway, because it won't only be an unread mention, but also an unread message, which should be shown in the counter anyway, right?

**Comment by RememberTheAir** [comment]
Yes tdesktop used to sum unread messages (or chats, based on your unread counter settings) and unread mentions from groups

**Comment by RememberTheAir** [comment]
unstale, still a different behavior from the mobile apps

**Comment by RememberTheAir** [comment]
can it be fixed? the problem dates back to two major releases ago

**Comment by RememberTheAir** [comment]
unstale :')

**Comment by RememberTheAir** [comment]
unstale

**Comment by 23rd** [comment]
@RememberTheAir Can you please to take a look at this debug win build?
https://github.com/telegramdesktop/tdesktop/actions/runs/17561255009

**Comment by RememberTheAir** [comment]
It looks like it works as intended. I mentioned an account in a group chat, and the sidebar shows the unread counter at "1", with these badge settings:

<img width="478" height="206" alt="Image" src="https://github.com/user-attachments/assets/c972fa41-cef1-4d77-b413-dbe16475858e" />

**Comment by 23rd** [comment]
Feel free to reopen.
